charts/edge-site/templates/service.yaml

```diff
@@ -12,5 +12,5 @@ spec:
       targetPort: 9099
     - name: metrics
       protocol: TCP
-      port: 6002
-      targetPort: 6002
+      port: 6001
```

Comment:
Does this require upgrading the app version?

Reply:
yeah, it will. good catch.